apps/furo/components/AddressInput.tsx

```diff
@@ -0,0 +1,68 @@
+import { isAddress } from '@ethersproject/address'
```

Comment:
There is already an Ens input for this exact purpose. Checkout apps/furo/components/stream/CreateForm

```
<Web3Input.Ens
  value={value}
  onChange={onChange}
  error={!!error?.message}
  placeholder="Address or ENS Name"
/>
```

Reply:
Aw crap, didn't notice, will replace, thanks

Reply:
Can also just make the whole modal wider ig... @chillichelli

Reply:
Hmmm, yeah first thought was truncate but that might not be the best solution in this particular case. Wider modal might not work well on mobile too so I guess smaller text is the only way

Reply:
ahh yeah the wallet/bento buttons should have the offset as well

Reply:
This looks good tho, love it!

Reply:
The loader could still be there tho but absolute positioned, and just hide when loading is done